### ISSUE TITLE
Add UI for user roles

### ISSUE THREAD

**Issue by andrixnet** [opencaching/opencaching-pl#2030]
The new user roles ( https://github.com/opencaching/opencaching-pl/issues/1727 ) are an interesting new feature, however managing them (by site admin) lacks any UI. 

I think the role assignments could be added in the Admin.UserAdmin page.

**Comment by kojoty** [comment]
I fully agree, but this is strictly internal so for now roles can be still managed by DB - there are so many much more urgent things... but let this issue wait for better day :)
